public/sw.js

```diff
@@ -0,0 +1,45 @@
+const pwaCache = 'pwa-cache-1'
+const staticCache = [
+      '/index.html',
+      '/index.css',
+      '/index.js',
+      '/logo4.png',
+      '/utilities.js',
+      '/RB.png',
+      '/LM.png',
+      '/KB.png',
+      '/location.svg'
+    ]
+
+self.addEventListener('install', (e) => {
```

Comment:
Service worker is crazy!! Super cool to see it working here and makes all the research we've done make alot more sense